prov/gni/include/gnix.h

```diff
@@ -508,6 +521,24 @@ struct gnix_fid_ep {
 	struct gnix_htd_pool htd_pool;
 	struct gnix_reference ref_cnt;
 	struct gnix_fid_ep_ops_en ep_ops;
+
+	struct fi_info *info;
+	struct fi_ep_attr ep_attr;
+	struct gnix_ep_name src_addr;
+
+	/* FI_EP_MSG specific. */
+	struct gnix_vc *vc;
+	int conn_fd;
+	int conn_state;
+	struct gnix_ep_name dest_addr;
```

Comment:
can we make this a pointer to a ```gnix_ep_name``` just trying to think of ways to avoid too much bloat in the size of ```gnix_fid_ep```.